### PR TITLE
[BD-26] Fix bug with end exam button and remove exam review text from submitted page

### PR DESCRIPTION
**Pull request by ViktorRusakov**
Fixed a bug where error would be displayed if user tried to click end my exam button when timer reached 00:00, also removed exam review text from submitted page

JIRA: 
- [EDUCATOR-5860](https://openedx.atlassian.net/browse/EDUCATOR-5860)
- [EDUCATOR-5866](https://openedx.atlassian.net/browse/EDUCATOR-5866)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @ViktorRusakov! I've created [BLENDED-913](https://openedx.atlassian.net/browse/BLENDED-913) to keep track of it in Jira.   More details are on the [BD-26](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2523627980/BD-26+Proctoring+MFE+and+configuration) project page.


When this pull request is ready, tag your edX technical lead.

File: src/data/thunks.js [hunk 1]
```diff
@@ -281,18 +281,36 @@ export function resetExam() {
...
       );
       return;
     }
+
+    const { attempt_id: attemptId, exam_url_path: examUrl } = activeAttempt;
+    if (!exam.attempt || attemptId !== exam.attempt.attempt_id) {
+      try {
+        await submitAttempt(attemptId);
+        window.location.href = examUrl;
+        if (useWorker) {
```
Comment (Zacharis278) [inline src/data/thunks.js:301]: this seems like repeated code from another place in this same function. Can we break this out into a helper or something?

**Review by Zacharis278 — CHANGES_REQUESTED** [review]
just a codestyle NIT, otherwise looks good.

File: src/data/thunks.js (hunk 1 above)
Comment (ViktorRusakov) [reply src/data/thunks.js]: updated in 3a09a06

**Review by Zacharis278 — APPROVED** [review]

**Pull request merged**